internal/gamepad/api_windows.go

```diff
@@ -198,7 +201,16 @@ func getRawInputDeviceList(pRawInputDeviceList *_RAWINPUTDEVICELIST, puiNumDevic
 }
 
 func setWindowLongPtrW(hWnd uintptr, nIndex int32, dwNewLong uintptr) (uintptr, error) {
-	h, _, e := procSetWindowLongPtrW.Call(hWnd, uintptr(nIndex), dwNewLong)
+	if procSetWindowLong == nil {
```

Comment:
This variable's name is odd...

IMO we don't need this global variable. Instead,

```
var p *windows.LazyProc
if procSetWindowLongPtrW.Find() == nil {
	// 64-Bit Windows.
	p = procSetWindowLongPtrW
} else {
	// 32-Bit Windows.
	p = procSetWindowLongW
}
h, _, e := p.Call(...
```

should be fine.

Reply:
If Find() fails, next call will load the DLL again. Is that OK with you?

Reply:
> If Find() fails, next call will load the DLL again. Is that OK with you?

Find() always fails on 32bit Windows, right?

Reply:
Oh right, this is only called once anyway. Fixing.